pkg/systemd/init.js

```diff
@@ -218,6 +227,31 @@ define([
                 });
         }
 
+        function wait_valid(proxy, callback, args) {
+            proxy.wait(function() {
+            if (proxy.valid)
+                callback(proxy, args);
+            });
+        }
+
+        function add_timer_properties(timer_unit,unit) {
+            unit.LastTriggerTime = moment(timer_unit.LastTriggerUSec/1000).fromNow();
```

Comment:
The problem with this is that it doesn't update. Using moment.js is very nice, but it does bring in the whole refresh question: If you use timestamps / specific times, you can just render them and that's it. If you compare to "now", then that needs to update or become confusing after a while.
Related to this: I would feel better if the times gathered from the backend were actual timestamps. Then you could run the moment.js conversion when displaying, in this case: https://github.com/cockpit-project/cockpit/pull/4503/files#diff-276d25fb4053326350efbc4f097771c2R291